cvat/apps/dataset_manager/util.py

```diff
@@ -15,7 +15,7 @@
 from datetime import timedelta
 from enum import Enum
 from threading import Lock
-from typing import Any
+from typing import Any, Iterable
```

Comment:
It's better to use `collections.abc` instead, `typing` symbols are deprecated.

https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes